rust/worker/src/types/segment.rs

```diff
@@ -24,6 +25,7 @@ impl From<SegmentType> for String {
             SegmentType::Record => "urn:chroma:segment/record".to_string(),
             SegmentType::Sqlite => "urn:chroma:segment/metadata/sqlite".to_string(),
             SegmentType::BlockfileMetadata => "urn:chroma:segment/metadata/blockfile".to_string(),
+            SegmentType::Metadata => "urn:chroma:segment/metadata".to_string(),
```

Comment:
i think "urn:chroma:segment/metadata/blockfile" is what we want to use? maybe im confused

Reply:
Oh yeah, that is what I want. While we're here, though, what exactly is this for? Is this a storage location of some sort or?